docs/general/ddl/data-types.rst

```diff
@@ -255,6 +256,54 @@ Inserting values that are either too short or too long results in an error:
     DROP OK, 1 row affected (... sec)
 
 
+.. _data-type-json:
+
+``json``
+--------
+
+A type representing a JSON string.
+
+This type primarily exists for compatibility with PostgreSQL. It cannot to be
```

Comment:
@jweberpm Maybe you could also have a look over the docs.

Reply:
I committed some changes to the docs. I think it needs to be more explicit that the JSON data type is only for typecasting.